.pre-commit-config.yaml

```diff
@@ -0,0 +1,47 @@
+repos:
+-   repo: https://github.com/psf/black
+    rev: 24.4.2
+    hooks:
+    - id: black
+      args: [
+              "--line-length=100",
+              "-t", "py39",
+              "-t", "py310",
+              "-t", "py311",
+            ]
+      files: frontend
+-   repo: https://github.com/PyCQA/isort
+    rev: 5.13.2
+    hooks:
+    - id: isort
+      args:
+        [
+          "--py",
+          "311",
+          "--profile",
+          "black",
+          "-l",
+          "100",
+          "-o",
+          "autoray",
+          "-p",
+          "./frontend",
+          "--skip",
+          "__init__.py",
+          "--filter-files",
+        ]
```

Comment:
Any chance these configs can be done based on the config files in the repo?

Reply:
not sure, but doesn't look like it from glancing at the docs.

Reply:
Do you know if the commands just run in the current working directory? Normally black and isort will automatically detect the nearest config file, which for us is the [pyproject.toml](https://github.com/PennyLaneAI/catalyst/blob/main/pyproject.toml) in the repo root.

Given that Catalyst's config is a bit different than the flags listed here, I think the config in this file is either redundant or incorrect, and I would prefer not to have divergent configurations used in different contexts.
Can you test if omitting all flags works as expected?